addon/components/polaris-date-picker.js

```diff
@@ -10,6 +11,8 @@ import {
   getNextDisplayMonth,
   getPreviousDisplayYear,
   getPreviousDisplayMonth,
+  weekdays, // obj
+  isSameDay,
 } from '../utils/dates';
 
 // TODO: add changes for Polaris v2.2.0
```

Comment:
How 'bout these guys? Are those changes present now?

Reply:
Ah yeah this can be removed. Ironically the additions from 2.0->2.2 are stripped out in 2.11 (other than translations on accessibility labels which we don't have)